### PR TITLE
Fixes #25687: Same as parent ticket for branch 8.1

### PR DESCRIPTION
**Pull request by clarktsiory**
https://issues.rudder.io/issues/25687

Add `nonEmpty` condition similarly to https://github.com/Normation/rudder/pull/5937/files, to return the list of group properties when there are no nodes

### CODE REVIEW

File: webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/GroupsApi.scala [hunk 1]
```diff
@@ -1367,61 +1369,68 @@ class GroupApiService14(
...
+          }
 
+          success ++ error
+        }
+      }
+      groupPropertyStatuses = checkedNodeProperties match {
+                                case Nil       => // no Node is group : just use resolved group properties
+                                  parentProperties match {
+                                    case s: SuccessNodePropertyHierarchy =>
+                                      s.resolved.map(InheritedPropertyStatus.from(_))
```
Comment (clarktsiory) [inline webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/GroupsApi.scala:1421]: I just realized we should check that the resolved property is "Displayed" before transforming it in 8.2 (the "Visibility" is only from 8.2 so it simplifies things here)

**Comment by clarktsiory** [comment]
Commit modified

**Review by fanf — APPROVED** [review]
seems to work as expected !

**Comment by clarktsiory** [comment]
OK, merging this PR

**Comment by clarktsiory** [comment]
OK, merging this PR

**Pull request merged**